From 45805626ea3f21abf3f67582c33578d9bf7bec5c Mon Sep 17 00:00:00 2001
From: Ding Wei <leo.ding@rock-chips.com>
Date: Thu, 12 Nov 2020 10:43:48 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable mpp relative codec

CONFIG_MPP_ROCKCHIP_RKVDEC2: rkv-decoder-v2
CONFIG_MPP_ROCKCHIP_RKVENC:  rkv-encoder-v1
CONFIG_MPP_ROCKCHIP_JPGDEC:  rkv-jpeg-decoder-v1

Change-Id: Ia98a2908caaaff24f5df4ec507590b276fc15ca5
Signed-off-by: Ding Wei <leo.ding@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 98ca8fe26e48..d62c9a0c7449 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -603,10 +603,13 @@ CONFIG_ROCKCHIP_RGA2=y
 CONFIG_IEP=y
 CONFIG_ROCKCHIP_MPP_SERVICE=y
 CONFIG_ROCKCHIP_MPP_RKVDEC=y
+CONFIG_ROCKCHIP_MPP_RKVDEC2=y
 CONFIG_ROCKCHIP_MPP_VDPU1=y
 CONFIG_ROCKCHIP_MPP_VEPU1=y
 CONFIG_ROCKCHIP_MPP_VDPU2=y
 CONFIG_ROCKCHIP_MPP_VEPU2=y
+CONFIG_ROCKCHIP_MPP_RKVENC=y
+CONFIG_ROCKCHIP_MPP_JPGDEC=y
 CONFIG_SOUND=y
 CONFIG_SND=y
 CONFIG_SND_DYNAMIC_MINORS=y
-- 
2.35.3

